### PR TITLE
Log the message payload before demultiplexing

### PR DESCRIPTION
**Pull request by elight**

### CODE REVIEW

**Comment by rwz** [comment]
LGTM, but I wonder about these CI failures. Were they caused by your change?

**Comment by elight** [comment]
Unknown. I'll have to investigate.

File: lib/mediators/messages/creator.rb [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+      }
     end
 
     def call
-      @message.save
+      Message.new(@args).save
+      Pliny.log(@args.merge(messages_creator: true, telex: true))
       Jobs::MessagePlex.perform_async(@message.id)
       Telex::Sample.count "messages"
       @message
```
Comment (elight) [inline lib/mediators/messages/creator.rb]: @rwz I made an oops. I got rid of `@message` on L4 so...

Comment (elight) [inline lib/mediators/messages/creator.rb]: That's what I get for not getting telex's dev env set up locally.

**Comment by rwz** [comment]
:+1:

**Comment by elight** [comment]
@jingweno Ready to deploy.

**Pull request merged**